std/assembly/builtins.ts

```diff
@@ -2336,4 +2336,11 @@ declare function trace(
 @external("env", "seed")
 declare function seed(): f64;
 
-/* eslint-enable @typescript-eslint/no-unused-vars */
+@final
+export declare class arguments {
```

Comment:
Is it possible to declare a namespace?  If not, that should be an added feature request.